src/swap/components/SwapProvider.tsx

```diff
@@ -93,6 +93,18 @@ export function SwapProvider({
     }
     // Emit Status
     onStatus?.(lifeCycleStatus);
+    // Reset lifecycle status after a success
+    if (
+      lifeCycleStatus.statusName !== 'init' &&
```

Comment:
we actually don't need that first one because it cannot be `init` if it is `success`